server/src/main/java/org/elasticsearch/common/Rounding.java

```diff
@@ -218,17 +218,20 @@ public Rounding build() {
     static class TimeUnitRounding extends Rounding {
 
         static final byte ID = 1;
+        static final long TZ_OFFSET_NON_FIXED = -1;
```

Comment:
comment nit: maybe include a small comment what `-1` indicates, and why this can never be -1 in a regular offset